source/includes/resources/records/_representation.md

```diff
@@ -0,0 +1,3 @@
+### Representation
+
+The Records ressource is used to search through published records.
```

Comment:
This section is a bit tricky since it's a very close duplication of what the deposit metadata representation looks like... On the other hand, it's a different serialization format (`application/vnd.zenodo.v1+json`), so there are some extra/different fields to document, e.g. the `stats` field is new and there's no `upload_type` (since it's exposed as `resource_type`). 

For now let's copy from deposit (without the metadata though, only the [top level object](https://developers.zenodo.org/#representation), and add documentation for the `stats` field.